packages/inference/src/lib/makeRequestOptions.ts

```diff
@@ -18,8 +18,6 @@ export async function makeRequestOptions(
 		stream?: boolean;
 	},
 	options?: Options & {
-		/** For internal HF use, which is why it's not exposed in {@link Options} */
-		includeCredentials?: boolean;
```

Comment:
Probably best not to remove this, in case other libraries depend on it.

Reply:
Ah, other libraries may use it. I couldn't find any usages in this repo so assumed it wasn't used.

Reply:
tbh, I'm not sure either 😅 (since it isn't exposed to end-users). I interpreted the comment as "used by other HF libraries". 🤷‍♂️